src/utils/flags/zyppflags.h

```diff
@@ -170,7 +191,7 @@ namespace ZyppFlags {
    * \returns The first index in argv that was not parsed
    * \throws ZyppFlagsException or any subtypes of it
    */
-  int parseCLI ( const int argc, char * const *argv, const std::vector<CommandGroup> &options, const int firstOpt = 1 );
+  int parseCLI (const int argc, char * const *argv, const std::vector<CommandGroup> &options, const int firstOpt = 1 );
```

Comment:
I don't see the rule for it ;)

In the 2nd hunk src/utils/flags/flagtypes.cc e.g. you add the missing WS before the closing ')' (to make it consistent with the `( `?).  Here it was consistent and you remove the WS.